5a/Client/src/Program.fs

```diff
@@ -82,17 +85,20 @@ let main _args =
     client.Connect
         (fun error ->
             task {
-                if not (error = null) then
-                    printfn $"Error connecting to cluster: ${error}"
-                    return false
-                else
-                    printfn "Connected to cluster"
-                    return true
+                Async.Sleep(TimeSpan.FromSeconds(1)) |> Async.RunSynchronously
```

Comment:
can we not run it asynchronously using the `!` syntax?

Reply:
https://github.com/PiotrJustyna/road-to-orleans/pull/312/commits/45ca9ff16c27150a2d489cc01c8761d137ac5969